zstacklib/zstacklib/utils/file_metadata_handler.py

```
@@ -275,6 +280,51 @@ def _do_cleanup(self, metadataPath):
+        if not os.path.isdir(metadataDir):
+            logger.debug("cleanup_all: metadataDir %s does not exist, nothing to clean", metadataDir)
+            return {}
+
+        meta_name_re = re.compile(
+            r'^[0-9a-f]{32}\.vmmeta(?:\.tmp|\.summary|\.summary\.tmp)?$')
+
+        removed_any = False
+        failures = []  # list of "path: cause"
+        try:
+            names = os.listdir(metadataDir)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)